### PR TITLE
Use namespace for dpdk PTF tests

### PR DESCRIPTION
**Pull request by Hoooao**
Previously the test driver `run-dpdk-ptf-test.py` modified the host's network config which caused side-effects like disabling IPv6. Using namespace, which would be cleaned up after the test, resolved this. Code is mostly from `run-bmv2-ptf-test.py`.

### CODE REVIEW

File: backends/dpdk/run-dpdk-ptf-test.py [hunk 1]
```diff
@@ -81,21 +85,12 @@
...
-
-
-# We want to disable IPv6 to avoid ICMPv6 spam
-def diable_IPv6():
-    testutils.exec_process("sysctl -w net.ipv6.conf.all.disable_ipv6=1")
-    testutils.exec_process("sysctl -w net.ipv6.conf.default.disable_ipv6=1")
-    # Also filter igmp packets, -w is necessary because of a race condition
-    testutils.exec_process("iptables -w -A OUTPUT -p 2 -j DROP")
-    return testutils.SUCCESS
+def is_port_alive(ns, port) -> bool:
```
Comment (fruffy) [inline backends/dpdk/run-dpdk-ptf-test.py:88]: Is there a more pythonic way to do this? Something where we do not have to call netstat?

For example, like this check:
https://github.com/p4lang/p4c/blob/main/tools/testutils.py#L134

**Review by fruffy — APPROVED** [review]

File: backends/dpdk/run-dpdk-ptf-test.py (hunk 1 above)
Comment (fruffy) [reply backends/dpdk/run-dpdk-ptf-test.py:88]: We have utilities for this. 
`bridge.ns_exec` should take care of executing in the right namespace.

Comment (Hoooao) [reply backends/dpdk/run-dpdk-ptf-test.py:88]: `ns_exec` simply appends `ip netns exec <ns>` before the cmd. It cannot handle calling python functions in an easy way.

Comment (fruffy) [reply backends/dpdk/run-dpdk-ptf-test.py:88]: Oh I see what you mean, okay let's leave it like this then. Just use `bridge.ns_exec` instead.

**Comment by jafingerhut** [comment]
Merging this.

**Pull request merged**